lib/tests/misc.nix

```
@@ -196,6 +196,25 @@ runTests {
     expected = "hellohellohellohellohello";
   };
 
+  testTrimString = {
+    expr = mapAttrs strings.trim {
```

```suggestion
    expr = mapAttrs (_: strings.trim) {
```
The function passed to `mapAttrs` takes two args (attribute name and value, respectively). By the way, you can run the tests in a local clone with `nix-build lib/tests/release.nix`.

---

> By the way, you can run the tests in a local clone with `nix-build lib/tests/release.nix`.

Thanks!